fatrace.c

```diff
@@ -645,6 +645,9 @@ parse_args (int argc, char** argv)
         switch (c) {
             case 'C':
                 option_comm = strdup (optarg);
+		if (!option_comm) {
+			errx(EXIT_FAILURE, "memory allocation failed for -- command");
+		}
```

Comment:
Please fix the indentation and the option name (no space).  Note that you can use `err()`, as strdup sets errno properly.

Please also apply this check to `option_output`. In bigger programs I usually introduce a `strdupx()` helper function which does that check and crashes the program.